python/cudf/cudf/core/column/categorical.py

```diff
@@ -1161,6 +1161,9 @@ def _mimic_inplace(self, other_col, inplace=False):
 
         return out
 
+    def view(self, dtype):
+        raise NotImplementedError('Can not create a view of a categorical column')
```

Comment:
Is this absolute, or should we reword this to say something like "categorical column views are currently unsupported." ?